core/src/main/java/org/apache/calcite/sql/SqlWithItem.java

```diff
@@ -92,10 +92,12 @@ private static class SqlWithItemOperator extends SqlSpecialOperator {
         int leftPrec,
         int rightPrec) {
       final SqlWithItem withItem = (SqlWithItem) call;
+
```

Comment:
Random formatting changes. Please remove since this isn't a file we're directly touching.

Reply:
Done